tests/test_sourmash.py

```diff
@@ -204,7 +204,7 @@ def test_do_compare_output_csv():
         with open(os.path.join(location, 'xxx')) as fp:
             lines = fp.readlines()
             assert len(lines) == 3
-            assert lines[1:] == ['1.0,0.93\n', '0.93,1.0\n']
+            assert lines[1:] == [u'1.0,0.93\n', '0.93,1.0\n']
```

Comment:
Since this is breaking in Python 2.7, might as well parse it as floats and then compare? Dealing with the `u''` strings is annoying =/

Reply:
yep.